### PR TITLE
Update .gitignore

### PR DESCRIPTION
**Pull request by karan2762002**
*Replace this paragraph with a description of what this PR is changing or adding, and why. Consider including before/after screenshots.*

*List which issues are fixed by this PR. You must list at least one issue.*

*If you had to change anything in the [flutter/tests] repo, include a link to the migration guide as per the [breaking change policy].*

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test-exempt.
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

**Comment by goderbauer** [comment]
@karan2762002 Can you please update the PR description with some context describing why you want to make this change? Also, can you please sign the CLA as directed by the bot above? Thanks.

File: .gitignore [hunk 1]
```diff
@@ -21,6 +21,7 @@
 .project
 .settings/
 .vscode/
+.dart/
```
Comment (goderbauer) [inline .gitignore:24]: @DanTup Does VS Code actually produce this directory? Should it be ignored?

Comment (DanTup) [reply .gitignore:24]: There's nothing I'm aware of in VS Code or the Dart/Flutter extensions for VS code that creates `.dart` directory.

**Comment by karan2762002** [comment]
> @karan2762002 Can you please update the PR description with some context describing why you want to make this change? Also, can you please sign the CLA as directed by the bot above? Thanks.

ya ofcourse

**Pull request closed without merging**